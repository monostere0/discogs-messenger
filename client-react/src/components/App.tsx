import * as React from 'react';
import RouteProvider from '../RouteProvider';

export default class App extends React.Component {
  render() {
    return <RouteProvider />;
  }
}
