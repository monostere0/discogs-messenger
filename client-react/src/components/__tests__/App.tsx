import * as React from 'react';
import * as renderer from 'react-test-renderer';

import App from '../App';

describe('src/App', () => {
  it('should render', () => {
    const tree = renderer.create(<App />);
    expect(tree).toMatchSnapshot();
  });
});