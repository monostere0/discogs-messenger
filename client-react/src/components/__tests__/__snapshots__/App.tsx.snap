// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/App should render 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.tsx
    </code>
     and save to reload.
  </p>
</div>
`;
